aim/api/resource.py

```diff
@@ -1223,6 +1223,7 @@ class SecurityGroupRule(AciResourceBase):
         ('icmp_type', t.string()),
         ('icmp_code', t.string()),
         ('remote_group_id', t.name),
+        ('tDn', t.string(256)),
```

Comment:
Would an empty string make sense as a default value for this field?

Reply:
It is only set by GBP, and Sayali sets it to empty string I believe, when it is not updated. I can add it to the init as well.